test/parallel/test-buffer-indexof.js

```diff
@@ -81,43 +81,54 @@ assert.equal(Buffer.from('ff').indexOf(Buffer.from('f'), 1, 'ucs2'), -1);
 // test hex encoding
 assert.equal(
     Buffer.from(b.toString('hex'), 'hex')
-    .indexOf('64', 0, 'hex'), 3);
+    .indexOf('64', 0, 'hex'),
```

Comment:
@targos I've made an effort to enforce indentation in cases like this and submitted the PR upstream. https://github.com/eslint/eslint/pull/5940